API.md

```diff
@@ -34,7 +38,14 @@ Used to register models, knex instances, and migration directory information on
 Returns the knex instance used by `request.route`'s plugin. In other words, this returns the knex instance for `request.route`'s active realm, falling back to the root server's server-wide default knex instance.
 
 #### `request.models([all])`
-Returns an object containing models keyed by `name`.  When `all` is `true`, models across the entire server are returned.  Otherwise, only models declared within `request.route`'s plugin (or, active realm) are returned.
+Returns an object containing models keyed by `name`.  When `all` is `true`, models across the entire server are returned. Otherwise, only models declared within a.) `request.route`'s plugin (or, active realm) and b.) any plugins for which `request.route`'s plugin is an ancestor (e.g. if `request.route`'s plugin has registered a plugin that registers models) are returned.
+
+### Response toolkit decorations
+#### `h.knex()`
+Returns the knex instance used by the current route's plugin. In other words, this returns the knex instance for the active realm as identified by `h.realm`, falling back to the root server's server-wide default knex instance.
```

Comment:
Let's chat about this and find a good way to describe the new behavior 👍

Reply:
👌 that'd be great. thanks, my dude. i'll bug ya